ONSdigital/design-system
# Fix: Use semantically correct element for inputs with prefix/suffix

### What is the context of this PR?
Fixes #2137 

- Sets a `span` for the prefix/suffix of an input if no long form `title` for the abbreviated visible text is set.
- Added new macro input tests to check for these
- Added missing Axe tests for input macro
- Updated mutually exclusive JS to not rely on `title` attribute to select prefix/suffix label

### How to review
- Check inputs with prefix/suffix only use the `abbr` element if a `title` is set. Otherwise a `span` should be used.
- Check new tests
